### DISCUSSION TITLE
visits: 3 [supervised] -10

### DISCUSSION THREAD

**Discussion by matt873** [abrahamjuliot/creepjs#237] [Q&A]
What does it mean? I am not using any proxy and my FP is normal.
<img width="258" alt="anty_xkMYT0A4ze" src="https://github.com/abrahamjuliot/creepjs/assets/108935493/5f742a2c-9d5f-477a-bb74-ec80972557d1">

**Comment by abrahamjuliot** [comment]
It's normal. Most of my browsers are supervised too. 

It basically means that none or a tiny fraction of site visitors possess the same degree of entropy as your browser during a seven-day period. To put it differently, our browsers exhibit a remarkable degree of uniqueness, and the code has designated distinct areas of the fingerprint for training and observation before granting a higher level of trust. 

Currently, there is only one focus area of entropy, but we may add more (each of these are secrets - for now).

For genuine fingerprints, 66% can blossom to 90%+ over a short period of time.

**Comment by matt873** [comment]
Thank you for your reply. I haven't done any canvas masking or tweaked any other parameters of my browser. So was surprised it may be due to uniqueness.
